dacScanComparison2.py

```diff
@@ -0,0 +1,77 @@
+#
+#
+# This macro takes the name of the chamber and all the scandates from the scandates list file and produces tables for the twelve parameter of the VFAT electronics
+#
+#
+
+def getcolumn(filename, column, vfat):
+    f=open(filename,"r")
+    lines=f.readlines()
+    values=[]
+    if(vfat==-1):
+        for line in lines:
+            values.append(line.strip().split(' ')[column])
+    else:
+        values = "N.A."
+        for line in lines:
+            if(line.strip().split('\t')[0]==vfat):
+                values = line.strip().split('\t')[column]
+    f.close()
+    #print values
+    return values
+
+import argparse
+import os,commands
+import sys
+usage = 'python dacScanComparison.py python dacScanComparison.py /path/to/DACscans/ --sdlist scandatelist.txt --vfat all -o /path/for/the/results/'
+parser = argparse.ArgumentParser(usage)
```

Comment:
This should not be a usage string, and since you have not defined a keyword this is interpreted as the `prog` argument of the `ArgumentParser` constructor, see:

https://docs.python.org/2/library/argparse.html#argparse.ArgumentParser

Additionally it's not necessary to provide a usage string since this will automatically be determined by `ArgumentParser` based on the arguments you define with `add_argument` or other class methods.

Typically this should be a `description`.